extensions/amp-accordion/1.0/amp-accordion.js

```diff
@@ -113,25 +128,54 @@ function getState(element, mu) {
 }
 
 /**
+ * @param triggerEvent
+ * @param triggerEvent
+ * @param triggerEvent
+ * @param triggerEvent
  * @param {!Element} sectionElement
  * @param {!AccordionDef.SectionProps} props
+ * @param root0
+ * @param root0.expanded
+ * @param root0.children
+ * @param root0
+ * @param root0.expanded
+ * @param root0.children
+ * @param root0
+ * @param root0.expanded
+ * @param root0.children
+ * @param root0
+ * @param root0.expanded
+ * @param root0.children
+ * @param triggerEvent.expanded
+ * @param triggerEvent.children
  * @return {PreactDef.Renderable}
  */
-function SectionShim(sectionElement, {expanded, children}) {
+function SectionShim(triggerEvent, sectionElement, {expanded, children}) {
   useLayoutEffect(() => {
     toggleAttribute(sectionElement, 'expanded', expanded);
+
+    if (triggerEvent) {
```

Comment:
This shouldn't go via an effect. Instead we should supply `onExpanded/Collapsed` event handlers as the Preact props. This would probably also fix the issue with event triggering on mount.